Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.0.1'
```

Comment:
I'm all for new, but the version should still be defined here. This will force the dependency, so you can guarantee potential users have all of the features you expect to exist in the framework. I recommend a `~>` instead though